### ISSUE TITLE
Allow to load bitbloq *.xml files

### ISSUE THREAD

**Issue by javacasm** [vrruiz/visualino#11]
The format seems to be the same,  only need to include the *.xml extension in the load dialog

**Comment by vrruiz** [comment]
Yes, the same format, although we have to be careful with the blocks (until
now we are in sync).

**Comment by javacasm** [comment]
Ok, maybe using a defensive checking preventing crash from an undefined block

**Comment by vrruiz** [comment]
File filter added in github.
